### ISSUE TITLE
IOError when importing beholder

### ISSUE THREAD

**Issue by mrlzla** [chrisranderson/beholder#50]
There is some error occured when importing the module:
```
Traceback (most recent call last):
  File "train.py", line 28, in <module>
    from beholder.beholder import Beholder
  File "/usr/local/lib/python2.7/dist-packages/beholder/beholder.py", line 10, in <module>
    from beholder import im_util
  File "/usr/local/lib/python2.7/dist-packages/beholder/im_util.py", line 79, in <module>
    colormaps = np.load('{}/colormaps.npy'.format(resources_path()))
  File "/usr/local/lib/python2.7/dist-packages/numpy/lib/npyio.py", line 370, in load
    fid = open(file, "rb")
IOError: [Errno 2] No such file or directory: '/usr/local/lib/python2.7/dist-packages/beholder/resources/colormaps.npy'
```
Python: 2.7
TF version: 1.2.0
OS: Ubuntu 16.04
Bazel: 0.5.3
Pip: 9.0.1

**Comment by chrisranderson** [comment]
What command exactly did you use to `pip install`? When I run the following:

```python
import beholder
print(beholder.__path__)
```

I get the directory that I cloned Beholder to, not `/usr/local/lib/python2.7/dist-packages/beholder`. For a quick fix, you might be able to download the files that are missing from [the resources directory](https://github.com/chrisranderson/beholder/tree/master/beholder/resources) and place them where Python is looking for them. So after you download `colormaps.npy` from the page I just linked to, put it in `/usr/local/lib/python2.7/dist-packages/beholder/resources` and let me know how that goes. You might have to move some more things to be in the right place.

I'm thinking this is my problem, but I don't have time to fix it right now. I hope that lets you get up and running.

**Comment by mrlzla** [comment]
Yes, it worked good when I copied missing files.
Actually I run the command via sudo:
```
sudo pip install .
```
Can it be the reason for the error?

**Comment by chrisranderson** [comment]
I always do that too (which I guess is bad but I'm not sure why). There's some stuff I can put in `setup.py` that relates to data associated with the module - I'll have to look at that another time. I'll leave this issue open until I figure that out.

Glad it worked! :)

**Comment by wchargin** [comment]
Only slightly tangential…

In my experience, you'll be happiest with pip if you _always_ use a virtualenv. Using `sudo pip install` once often means that you'll have to always use that from now on to get the permissions to align, but you don't want to do that: you want to own the files, and you want any configuration done by post-install scripts to apply to your user, not `root`. (Also, running arbitrary post-install code under root isn't the best idea.)

If you really don't want to use a virtualenv, then `pip install --user PACKAGE` is the way to go. (But I honestly don't see a reason to do this: personally, I have zero packages installed outside of a virtualenv.)

**Comment by chrisranderson** [comment]
This should be fixed by ceefdba88595e3f91756d1db76c9eed25e6ca577. Thanks for reporting, @mrlzla!
